### ISSUE TITLE
API call "commit" not accepting "custom_version_major", "custom_version_minor" anymore?

### ISSUE THREAD

**Issue by litzenbergerm** [danmunn/redmine_dmsf#1452]
After migration to 3.0.12 the file upload via the API commit call seems not to accept the "custom_version_major", "custom_version_minor" fields anymore, when I upload a new version of an existing file?

Regardless of what "custom_version_major", "custom_version_minor" I pass with the call the version of the committed file stays with the previous version of the file.

        
       mycommit = { 'attachments': 
            {'folder_id': folderid,
            'uploaded_file': {
                'token': token,
                'version': '3',
                'custom_version_major': myrev[0],
                'custom_version_minor': myrev[1],
                'name': fname,
                'title': title,
                'description': description,
                'comment': comment 
        }}}
        uurl = self.url + 'projects/'+project+'/dmsf/commit.json'
        myfields = json.dumps(mycommit)
        resp = self.http.request('POST', 
    	       uurl, 
    	       body = myfields.encode('utf-8'),
    	       headers = self.hd) 

Environment:
  Redmine version                5.0.5.stable
  Ruby version                   3.0.2-p107 (2021-07-07) [x86_64-linux-gnu]
  Rails version                  6.1.7.2
  Environment                    production
  Database adapter               Mysql2
  Mailer queue                   ActiveJob::QueueAdapters::AsyncAdapter
  Mailer delivery                smtp
Redmine settings:
  Redmine theme                  Ait (includes JavaScript)
SCM:
  Subversion                     1.14.1
  Git                            2.34.1
  Filesystem                     
Redmine plugins:
  redmine_dmsf                   3.0.12

**Comment by picman** [comment]
I think that there is a mistake in the wiki. There should be _version_major_ and _version_minor_ instead of _custom_version_major_ and _custom_version_minor_. Could you re-try it with _version_major_ and _version_minor_?

**Comment by picman** [comment]
I think too, that `'version': '3',` makes no sense.

**Comment by litzenbergerm** [comment]
Ok, that fixed it! Thank you for the fast reaction!

However, I am pretty sure the code give in the issue was working for 3
years on a previous version of DMSF (can't recall the exact ver. now).

Am Do., 25. Mai 2023 um 08:14 Uhr schrieb Karel Pičman <
***@***.***>:

**Comment by picman** [comment]
Yeah, I have probably changed that in the code and omitted to update the wiki. It isn't mentioned in the changelog either, sorry. Wiki updated.
